opsdroid/events.py

```diff
@@ -100,13 +101,15 @@ class Event(metaclass=EventMetaClass):
 
     def __init__(
         self,
+        user_id=None,
         user=None,
         target=None,
         connector=None,
         raw_event=None,
         event_id=None,
         linked_event=None,
     ):  # noqa: D107
+        self.user_id = user_id
```

Comment:
I have one question over here should I do it like this
```
        user=None,
        target=None,
        connector=None,
        raw_event=None,
        event_id=None,
        linked_event=None,
        user_id=None,
```
Or I need to do changes in all test

Reply:
This change needs to happen here in the base class as well as in all the tests and places where events are created.

The big thing you need to do is go through the connectors and make sure they specify this keyword argument when they construct events having relieved them. i.e. [here](https://github.com/opsdroid/opsdroid/blob/master/opsdroid/connector/matrix/create_events.py) for matrix and [here](https://github.com/opsdroid/opsdroid/blob/master/opsdroid/connector/gitter/__init__.py#L81) for gitter. There should likely be a change in every connector.